scripts/action.js

```diff
@@ -331,6 +331,7 @@ function actionLoop(el){
 		localStorage.setItem("inMiddleOfAction", "false");
```

Comment:
The edits to action.js, output.js, overlayScreen.js, and action.html all result from a merge with master that got lumped in with the original commit. Very sorry about this! I am still learning things about Git and, like I said, messed up on some rollbacks.